### PR TITLE
Update model-libraries.ts and add count statistics for glyph-byt5 (https://arxiv.org/abs/2403.09622)

### PR DESCRIPTION
**Pull request by lzy-tony**

### CODE REVIEW

File: packages/tasks/src/model-libraries.ts [hunk 1]
```diff
@@ -214,6 +214,15 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 			term: { path: "gliner_config.json" },
 		},
 	},
+	glyph-byt5: {
```
Comment (osanseviero) [inline packages/tasks/src/model-libraries.ts:217]:
```suggestion
	"glyph-byt5": {
```

File: packages/tasks/src/model-libraries.ts [hunk 2]
```diff
@@ -214,6 +214,15 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
...
+		prettyLabel: "Glyph-ByT5",
+		repoName: "Glyph-ByT5",
+		repoUrl: "https://github.com/AIGText/Glyph-ByT5",
+		filter: false,
+		countDownloads: {
+			term: { path: "checkpoints/byt5_model.pt" },
```
Comment (osanseviero) [inline packages/tasks/src/model-libraries.ts:223]: I think https://huggingface.co/GlyphByT5/Glyph-SDXL-v2/tree/main follows a different structure unfortunately

File: packages/tasks/src/model-libraries.ts (hunk 1 above)
Comment (osanseviero) [inline packages/tasks/src/model-libraries.ts:217]: Some repos will need to have `library_name: glyph-byt5` before we merge this

**Review by osanseviero — COMMENTED** [review]
Very nice PR!

File: packages/tasks/src/model-libraries.ts (hunk 1 above)
Comment (lzy-tony) [reply packages/tasks/src/model-libraries.ts:217]: Hi, thanks for the review, I have added library name in https://huggingface.co/GlyphByT5/Glyph-SDXL-v2/tree/main

File: packages/tasks/src/model-libraries.ts [hunk 3]
```diff
@@ -214,6 +214,15 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 			term: { path: "gliner_config.json" },
 		},
 	},
+	"glyph-byt5": {
+		prettyLabel: "Glyph-ByT5",
+		repoName: "Glyph-ByT5",
+		repoUrl: "https://github.com/AIGText/Glyph-ByT5",
+		filter: false,
+		countDownloads: {
+			term: { path: "checkpoints/byt5_model.pt" },
```
Comment (osanseviero) [inline packages/tasks/src/model-libraries.ts:223]: Overall LGTM. Just note that we strongly suggest using conventions that will make it easier to scale up. For example, if you change the checkpoint name, this query won't work. To handle this, we usually use a file such as `config.json` or another query file that will always be there for all repos under a library. Would you consider doing something like this for the glyph repos?

Comment (lzy-tony) [reply packages/tasks/src/model-libraries.ts:223]: Thanks for the advice! We would consider such a format in the future, but at the moment we would prefer keeping the current format.

Comment (osanseviero) [reply packages/tasks/src/model-libraries.ts:223]: SG!

**Review by osanseviero — APPROVED** [review]
LGTM!

**Comment by lzy-tony** [comment]
Hi, do we still need more reviews for this pr to merge?

**Comment by osanseviero** [comment]
LGTM! Let's go :rocket:

**Comment by lzy-tony** [comment]
It seems like the matching rules have been changed before merging (https://github.com/huggingface/huggingface.js/pull/749), so unfortunately this pr might need to be updates :(

**Pull request merged**